src/web-blocks/image.tsx

```diff
@@ -36,17 +35,66 @@ const Config = {
   category: "core",
   icon: ImageIcon,
   group: "media",
-  props: {
-    styles: Styles({ default: "" }),
-    image: Image({
-      title: "Image",
-      default: "https://fakeimg.pl/400x200?text=Choose&font=bebas",
-    }),
-    alt: SingleLineText({ title: "Alt text", default: "", ai: true, i18n: true }),
-    width: SingleLineText({ title: "Width", default: "" }),
-    height: SingleLineText({ title: "Height", default: "" }),
-    lazyLoading: Checkbox({ title: "Lazy Load", default: true }),
-  },
+  // props: {
+  //   styles: Styles({ default: "" }),
+  //   image: Image({
+  //     title: "Image",
+  //     default: "https://fakeimg.pl/400x200?text=Choose&font=bebas",
+  //   }),
+  //   alt: SingleLineText({ title: "Alt text", default: "", ai: true, i18n: true }),
+  //   width: SingleLineText({ title: "Width", default: "" }),
+  //   height: SingleLineText({ title: "Height", default: "" }),
+  //   lazyLoading: Checkbox({ title: "Lazy Load", default: true }),
+  // },
+  ...registerChaiBlockSchema({
+    properties: {
+      styles: StylesProp(""),
+      image: {
+        type: "string",
+        title: "Image",
+        default: "https://fakeimg.pl/400x200?text=Choose&font=bebas",
+        ui: {
+          "ui:widget": "image",
+          "ui:options": {
```

Comment:
what does `ui:options` do?

Reply:
I was trying something. So I used that, there is no such thing in the docs. It was autosuggested by cursor.